distributed/diagnostics/tests/test_graph_layout.py

```diff
@@ -99,3 +99,14 @@ async def test_unique_positions(c, s, a, b):
 
     y_positions = [(gl.x[k], gl.y[k]) for k in gl.x]
     assert len(y_positions) == len(set(y_positions))
+
+
+@gen_cluster(client=True)
+async def test_layout_with_priorities(c, s, a, b):
+    gl = GraphLayout(s)
+    s.add_plugin(gl)
+
+    low = c.submit(inc, 1, key="low", priority=1)
```

Comment:
When would task priority be `None`, and how do I simulate that? If you have an answer, let me know and I'll add a test case.